### PR TITLE
Add documentation for custom blocks and custom skulls

### PR DESCRIPTION
**Pull request by Kas-tle**
Preview at:
https://pages.kastle.dev/geyser/custom-blocks/
https://pages.kastle.dev/geyser/custom-skulls/

Depends on https://github.com/GeyserMC/Geyser/pull/3505

### CODE REVIEW

**Comment by Chew** [comment]
Marking as draft until associated PR is merged

**Review by onebeastchris — APPROVED** [review]

File: _docs/geyser/custom-blocks.md [hunk 1]
```diff
@@ -0,0 +1,473 @@
+---
+layout: page
```
Comment (onebeastchris) [inline _docs/geyser/custom-blocks.md:2]: layout and permalink can be removed, they're no longer used
```suggestion
```

File: _docs/geyser/custom-blocks.md [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+title: Custom blocks with Geyser
+permalink: /geyser/custom-blocks/
```
Comment (onebeastchris) [inline _docs/geyser/custom-blocks.md:4]:
```suggestion
```
Set in toc.yml instead.

File: _docs/geyser/custom-skulls.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+---
+layout: page
```
Comment (onebeastchris) [inline _docs/geyser/custom-skulls.md:2]:
```suggestion
```

File: _docs/geyser/custom-skulls.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+title: Custom skulls with Geyser
+permalink: /geyser/custom-skulls/
```
Comment (onebeastchris) [inline _docs/geyser/custom-skulls.md:4]:
```suggestion
```

**Review by onebeastchris — CHANGES_REQUESTED** [review]
oops, forgot about these; Jens pointed them out

File: _docs/geyser/custom-blocks.md (hunk 1 above)
Comment (Kas-tle) [reply _docs/geyser/custom-blocks.md]: https://github.com/GeyserMC/GeyserWiki/pull/115/commits/9066dbd42a7d8e8f9fe2e3a0ed5f7f2c993c6cec#diff-6907c07388b3f34cb8c388adc86d69766e7681b013ea0bf58fe99988fe657ee7L2

File: _docs/geyser/custom-skulls.md (hunk 3 above)
Comment (Kas-tle) [reply _docs/geyser/custom-skulls.md]: https://github.com/GeyserMC/GeyserWiki/pull/115/commits/9066dbd42a7d8e8f9fe2e3a0ed5f7f2c993c6cec#diff-6907c07388b3f34cb8c388adc86d69766e7681b013ea0bf58fe99988fe657ee7L4

File: _docs/geyser/custom-blocks.md (hunk 2 above)
Comment (Kas-tle) [reply _docs/geyser/custom-blocks.md]: https://github.com/GeyserMC/GeyserWiki/pull/115/commits/9066dbd42a7d8e8f9fe2e3a0ed5f7f2c993c6cec#diff-6907c07388b3f34cb8c388adc86d69766e7681b013ea0bf58fe99988fe657ee7L4

**Review by onebeastchris — APPROVED** [review]

File: _docs/geyser/custom-skulls.md (hunk 3 above)
Comment (Kas-tle) [reply _docs/geyser/custom-skulls.md]: https://github.com/GeyserMC/GeyserWiki/pull/115/commits/9066dbd42a7d8e8f9fe2e3a0ed5f7f2c993c6cec#diff-5ad217849a96a4f6e0d4e9dbfa9e72060f812da7778302d65dd6129133bf5b93L2

File: _docs/geyser/custom-skulls.md (hunk 4 above)
Comment (Kas-tle) [reply _docs/geyser/custom-skulls.md]: https://github.com/GeyserMC/GeyserWiki/pull/115/commits/9066dbd42a7d8e8f9fe2e3a0ed5f7f2c993c6cec#diff-5ad217849a96a4f6e0d4e9dbfa9e72060f812da7778302d65dd6129133bf5b93L4

**Review by Jens-Co — APPROVED** [review]
Lgtm

**Pull request merged**